health-check/src/uyuni_health_check/config_loader.py

```diff
@@ -0,0 +1,66 @@
+import os
+import configparser
+from pathlib import Path
+import yaml
+import json
+from jinja2 import Environment, FileSystemLoader
+
+class ConfigLoader:
```

Comment:
IMO there is no need for this class, the things it does are generally unrelated.